examples/non-action-cable-js-client-with-lucky.html

```diff
@@ -0,0 +1,90 @@
+<!-- Open in a web browser with Lucky running -->
```

Comment:
This is great! The less JavaScript the better!

Reply:
why u no like JS?! lol